engine/runtime-integration-tests/src/test/java/org/enso/interpreter/test/AtomInteropTest.java

```diff
@@ -0,0 +1,146 @@
+package org.enso.interpreter.test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsInAnyOrder;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.notNullValue;
+
+import org.enso.test.utils.ContextUtils;
+import org.junit.Test;
+
+/**
+ * Tests various {@link com.oracle.truffle.api.interop.InteropLibrary interop} contracts for {@link
+ * org.enso.interpreter.runtime.data.atom.Atom atoms}.
+ */
+public class AtomInteropTest {
+
+  @Test
+  public void atomMembersAreConstructorFields_SingleConstructor() {
+    try (var ctx = ContextUtils.createDefaultContext()) {
+      var myTypeAtom =
+          ContextUtils.evalModule(
+              ctx,
+              """
+          type My_Type
+              Cons field_1 field_2
+
+          main =
+              My_Type.Cons 1 2
+          """);
+      assertThat(myTypeAtom.hasMembers(), is(true));
+      var memberNames = myTypeAtom.getMemberKeys();
+      assertThat("Has two fields", memberNames.size(), is(2));
+      assertThat(
+          "Member names are not qualified", memberNames, containsInAnyOrder("field_1", "field_2"));
```

Comment:
This test fails. The actual `memberNames` are reported as `["My_Type.field_1", "My_Type.field_2"]`. This is a regression that I have introduced in https://github.com/enso-org/enso/pull/9692/files#diff-be716397db895edb908afaa797100a7c7c39cfc2cee505b637f9494a84294adaR156-R162. Can be simly fixed.

Reply:
Fixed with https://github.com/enso-org/enso/pull/10661/commits/d34c3e721292c2d567f5a958aac36a0fd026c71a